src/bosh_azure_cpi/lib/cloud/azure/models/res_object_id.rb

```diff
@@ -13,5 +13,9 @@ def self.parse_with_resource_group(id_str, default_resource_group_name)
     def resource_group_name
       @id_hash[RESOURCE_GROUP_NAME_KEY]
     end
+
+    def to_s
```

Comment:
what is purpose of this function?

Reply:
supply one way for all the sub-class, say do logging and so on.